bin/node/runtime/pangolin/src/lib.rs

```diff
@@ -949,7 +957,13 @@ fn migrate_treasury() {
 				bond: old_proposal.ring_bond,
 			};
 
+			ring_proposals_count += 1;
+
 			migration::put_storage_value(NEW_PREFIX, b"Proposals", &hash, new_proposal);
+
+			if approvals.contains(&index) {
+				ring_approvals.push(index);
+			}
 		}
 		if old_proposal.kton_value != 0 {
```

Comment:
Do we need to check `ring_bond` field?

Reply:
Why

Reply:
O, typo mistake. I want to say `kton_bond`. I am not sure if there are proposals kton_value = 0, kton_bond !=0.

Reply:
Check value is okay.